CleverPush/Source/CPAppBannerModuleInstance.m

```diff
@@ -1088,7 +1088,7 @@ - (NSArray *)sortArrayByDateAndAlphabet:(NSArray *)array {
     NSArray *sortedArray = [array sortedArrayUsingComparator:^NSComparisonResult(id bannerElement1, id bannerElement2) {
         NSDate *bannerStartdate1;
         NSDate *bannerStartdate2;
-        NSComparisonResult result;
+        NSComparisonResult result = NSOrderedSame;
```

Comment:
This change in logic is intended?

Reply:
No logic is intended. I have tested with multiple banners.
It's just a minor adjustment to avoid decrement warnings.